pkg/apis/pipelinesascode/v1alpha1/types.go

```diff
@@ -70,11 +70,12 @@ type TaskInfos struct {
 
 // RepositorySpec is the spec of a repo
 type RepositorySpec struct {
-	ConcurrencyLimit *int         `json:"concurrency_limit,omitempty"`
-	URL              string       `json:"url"`
-	GitProvider      *GitProvider `json:"git_provider,omitempty"`
-	Incomings        *[]Incoming  `json:"incoming,omitempty"`
-	Params           *[]Params    `json:"params,omitempty"`
+	ConcurrencyLimit         *int         `json:"concurrency_limit,omitempty"`
+	URL                      string       `json:"url"`
+	GitProvider              *GitProvider `json:"git_provider,omitempty"`
+	Incomings                *[]Incoming  `json:"incoming,omitempty"`
+	Params                   *[]Params    `json:"params,omitempty"`
+	GithubAppTokenScopeRepos []string     `json:"github_app_token_scope_repos,omitempty"`
```

Comment:
why is it still there, don't we have move it in settings?

Reply:
Removed